src/cljs/main/broadfcui/main.cljs

```diff
@@ -263,8 +271,6 @@
              (cond
                (not (contains? user-status :go))
                [auth/UserStatus {:on-success #(swap! state update :user-status conj :go)}]
-               (not (contains? user-status :tos))
-               [auth/TermsOfService {:on-success #(swap! state update :user-status conj :tos)}]
```

Comment:
Moved this to the very top of `LoggedIn` where it is displayed if a user has not accepted the ToS but is registered. User must exist in Sam before accepting the ToS, which necessitated the order change